<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ficheInstitution</class>
 <widget class="QDialog" name="ficheInstitution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_nom">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pays">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>73</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pays  :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pays">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimerInstitution">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bloquedInstitution">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Debloquer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sauvegarderInstitution">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nomInstitutionControl">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_paysInstitutionControl">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
